client/src/components/Draftail/Draftail.scss

```diff
@@ -206,10 +206,15 @@ $draftail-editor-font-family: $font-sans;
   border-width: 1px;
 }
 
+.Draftail-ComboBox__status {
+  padding-inline-start: theme('spacing.3');
+}
+
 .Draftail-ComboBox__option[aria-selected='true'] {
   color: theme('colors.secondary.DEFAULT');
   border-radius: theme('borderRadius.sm');
   background: transparent;
+  cursor: pointer;
```

Comment:
I see two problems here: 

- Since the `aria-selected` attribute is applied by JS, there’s a slight lag in switching over to this cursor, which is a bit distracting. I think we could instead set the cursor directly on all `.Draftail-ComboBox__option`.
- I believe this would be worth fixing straight in https://github.com/springload/draftail.

Reply:
oh nice, so you suggest i try to fix it from the draftail codebase @thibaudcolas

Reply:
Sorry, I don’t think it was correct for me to indicate both of those things as problems! Your current approach is a good improvement as-is, so I’ve merged it like that. 

It’d be better to do this as I suggested in the separate project directly but no need for you to worry about this, I’ll get that done as part of further refactoring.